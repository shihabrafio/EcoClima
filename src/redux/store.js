import { configureStore } from '@reduxjs/toolkit';
import CountryReducer from './countrySlice';

const store = configureStore({
  reducer: CountryReducer,
});

export default store;
