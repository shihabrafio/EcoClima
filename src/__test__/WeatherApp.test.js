import React from 'react';
import { render as rtlRender } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import MainPage from '../components/WeatherApp';

const render = (component) => rtlRender(
  <Provider store={store}>
    {component}
  </Provider>,
);

describe('MainPage', () => {
  it('render MainPage Components', () => {
    render(<MainPage />);
    expect(MainPage).toMatchSnapshot();
  });
});
