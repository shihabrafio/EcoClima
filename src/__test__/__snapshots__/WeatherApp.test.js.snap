// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countires render Countries Components 1`] = `[Function]`;

exports[`Countires render MainPage Components 1`] = `[Function]`;

exports[`MainPage render MainPage Components 1`] = `[Function]`;
